packages/stryker/src/logging/MultiAppender.ts

```diff
@@ -0,0 +1,20 @@
+import { LoggingEvent } from 'log4js';
+
+export interface RuntimeAppender {
+  (loggingEvent: LoggingEvent): void;
+}
+
+export class MultiAppender {
+
+  constructor(private appenders: RuntimeAppender[]) { }
+
+  append(loggingEvent: LoggingEvent) {
+    this.appenders.forEach(appender => appender(loggingEvent));
+  }
+}
+
+export function configure(config: { appenders: string[] }, _: any, findAppender: (name: string) => RuntimeAppender ): RuntimeAppender {
```

Comment:
Where is this used?

Reply:
It is used by log4js. See https://log4js-node.github.io/log4js-node/writing-appenders.html. I'll add a jsdoc explaining it.